src/main/java/net/snowflake/client/jdbc/SnowflakeFileTransferAgent.java

```diff
@@ -2145,7 +2145,7 @@ static public void uploadWithoutConnection(SnowflakeFileTransferConfig config)
           destFileName, (requireCompress ? "yes" : "no"), uploadSize);
 
       SnowflakeStorageClient initialClient =
-          storageFactory.createClient(stageInfo, 1, encMat);
+          storageFactory.createClient(stageInfo, 1, encMat, /*session = */ null);
```

Comment:
This function is uploadWithoutConnection(), so there is no connection/session to add here. Therefore, I input null.